source/_integrations/ituran.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Ituran
+description: Instructions on how to add Ituran to Home Assistant.
+ha_category:
+  - Car
+  - Device Tracker
+  - Presence detection
+ha_release: '2025.1'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@shmuelzon'
+ha_domain: ituran
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
```

Comment:
This is expected to link to the upstream product or service, see also: 

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-high-level-description